webapp/channels/src/components/system_notice/notices.tsx

```diff
@@ -24,187 +14,8 @@ import * as UserAgent from 'utils/user_agent';
-                return false;
-            }
-
-            if (analytics.TOTAL_USERS && analytics.TOTAL_USERS < USERS_THRESHOLD) {
-                return false;
-            }
-
-            if (license.IsLicensed === 'true' && license.Cluster === 'true') {
-                return false;
-            }
-
-            return true;
-        },
-    },
-    {
-        name: 'ie11_deprecation',
-        title: (
-            <FormattedMessage
-                id='system_notice.title'
-                defaultMessage='Notice from Mattermost'
-            />
-        ),
-        allowForget: false,
-        body: (
-            <FormattedMessage
-                id='system_notice.body.ie11_deprecation'
-                defaultMessage='Your browser, IE11, will no longer be supported in an upcoming release. <link>Find out how to move to another browser in one simple step</link>.'
-                values={{
-                    link: (msg: React.ReactNode) => (
-                        <ExternalLink
-                            href='https://forum.mattermost.com/t/mattermost-is-dropping-support-for-internet-explorer-ie11-in-v5-16/7575'
-                            location='system_notices'
-                        >
-                            {msg}
-                        </ExternalLink>
-                    ),
-                }}
-            />
-        ),
-        show: (serverVersion) => {
-            // Don't show the notice after v5.16, show a different notice
-            if (ServerVersion.isServerVersionGreaterThanOrEqualTo(serverVersion, '5.16.0')) {
-                return false;
-            }
-
-            // Only show if they're using IE
-            if (!UserAgent.isInternetExplorer()) {
-                return false;
-            }
-
-            return true;
-        },
-    },
-    {
-
-        // This notice is marked as viewed by default for new users on the server.
-        // Any change on this notice should be handled also in the server side.
-        name: 'GMasDM',
-        allowForget: true,
-        title: (
-            <FormattedMessage
-                id='system_notice.title.gm_as_dm'
-                defaultMessage='Updates to Group Messages'
-            />
-        ),
-        icon: (<InformationOutlineIcon/>),
-        body: (
-            <FormattedMessage
-                id='system_noticy.body.gm_as_dm'
-                defaultMessage='You will now be notified for all activity in your group messages along with a notification badge for every new message.{br}{br}You can configure this in notification preferences for each group message.'
-                values={{br: (<br/>)}}
-            />
-        ),
-        show: (serverVersion, config, license, analytics, currentChannel) => {
-            return currentChannel?.type === 'G';
-        },
-    },
-];
+// All legacy notices have been deprecated and removed.
+// Add new notices to this array as needed.
+const notices: Notice[] = [];
```

Comment:
@hmhealey I didn't remove the ability to show system notice and left it as an empty array to be used in the future if needed